README.md

```diff
@@ -1,8 +1,9 @@
 # etcd operator
-Kubernetes stable :
+unit/integration:
+[![Build Status](https://jenkins-etcd-public.prod.coreos.systems/view/operator/job/etcd-operator-unit-master/badge/icon)](https://jenkins-etcd.prod.coreos.systems/view/operator/job/etcd-operator-master-unit/)
+e2e (Kubernetes stable):
```

Comment:
And I think the job name is `etcd-operator-unit-master` not `etcd-operator-master-unit`